pkg/adapter/adapter.go

```diff
@@ -624,12 +670,47 @@ func (a *Adapter) GetTxs(ctx context.Context) ([][]byte, error) {
 		return nil, err
 	}
 
-	a.Metrics.TxsProposedTotal.Add(float64(len(resp.Txs)))
+	a.metrics.TxsProposedTotal.Add(float64(len(resp.Txs)))
 	return resp.Txs, nil
 }
 
 // SetFinal handles extra logic once the block has been finalized (posted to DA).
 // For a Cosmos SDK app, this is a no-op we do not need to do anything to mark the block as finalized.
```

Comment:
Can we update this comment? We should add some docs as well, that epochs events are published one block after DA publication.